### PR TITLE
feat(welcome-popup): redesign welcome popup

### LINKED ISSUE #43: Bug(user): Notification-popup cut the text on small resolutions
**Steps to reproduce:**
1. Open the Phonebook
2. Delete the localStorage (if you already opened the site)
3. Decrease your Browser-Window.

**Actual behavior:**
The text in the notification popup will be cut off if `<mat-item>` is used.

**Screenshots:**
[Video](https://i.gyazo.com/c08209d1be4d67913dd30ca86632c8ad.mp4)

**Expected behavior:**
The user can see the full text on lower resolutions.

**Other Information:**
This problem happens on every dialog window.

### PR DESCRIPTION
**Pull request by mschwrdtnr**
- style notification dialog
- fully responsive resolves #43
- add mixin for mat-icon
- use mixin in whole project

Partly res #118 

*Update*
- Hover effect disabled
- change border and header color to theme primary
- renamed classname in `interactive-attribute-field`

**Images:**
image
image
image

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
this does not resolve #118 as it only changes the design.

**Comment by DanielHabenicht** [comment]
I would like to wait until pr-preview is ready so I can have a look at it.

**Comment by mschwrdtnr** [comment]
> this does not resolve #118 as it only changes the design.

Sure, i wrote *partly resolves #118*.

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

**Comment by DanielHabenicht** [comment]
Kannst du den Abständen in den Karten nochmal auf den Grund gehen?
image
Das ist sehr gedrungen.

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 1]
```diff
@@ -1,53 +1,132 @@
...
+      <mat-card class="pb-card-call pb-border-color-primary">
+        <mat-card-header>
+          <mat-card-title><mat-icon [color]="'primary'">error_outline</mat-icon></mat-card-title>
+        </mat-card-header>
+        <mat-card-title>Aufruf an dich!</mat-card-title>
+        <mat-card-content>
+          <span>
+            Dies ist ein Open Source Projekt und lebt von seinen aktiven Mitgestaltern. Dabei gibt es verschiedene
+            Themen die du übernehmen kannst. Du könntest uns z.B. helfen indem du ein
+            <a href="https://github.com/T-Systems-MMS/phonebook/issues" target="_blank">Issue</a> bearbeitest, einen
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]:
```suggestion
            <a href="https://github.com/T-Systems-MMS/phonebook/issues" target="_blank">Issue</a> bearbeitest, am
```

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]: ist zeitloser

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 2]
```diff
@@ -1,53 +1,132 @@
...
+            User-Guide schreibst oder einfach nur beim Verbessern der Software hilfst, indem du neue
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]:
```suggestion
            User-Guide schreibst oder einfach nur beim Verbessern der Software hilfst indem du neue
```

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 3]
```diff
@@ -1,53 +1,132 @@
...
+            <a href="https://github.com/T-Systems-MMS/phonebook/issues/new/choose" target="_blank">Bugs einstellst</a>
+            und beschreibst.
+          </span>
+          <span>
+            Melde dich bei Interesse gern
+            <a href="https://github.com/T-Systems-MMS/phonebook/" target="_blank">im offiziellen Repo</a> oder starte
+            mit einem Blick auf unsere
+            <a href="https://t-systems-mms.github.io/phonebook/" target="_blank">Dokumentation</a>.
+            <b>Programmieren ist keine Voraussetzung um ein Teil der Community zu werden!</b>
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]: Kann man den Satz noch zentriert auf die nächste Zeile bringen?

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 4]
```diff
@@ -1,53 +1,132 @@
...
+      </mat-card>
+    </div>
+    <div class="pb-flex-row pb-card-container">
+      <!-- Profilbild -->
+      <mat-card class="pb-border-color-primary">
+        <mat-card-header>
+          <mat-card-title><mat-icon class="big-icon" [color]="'primary'">person</mat-icon></mat-card-title>
+        </mat-card-header>
+        <mat-card-title>
+          Das Profilbild kann nun von dir selbst ausgetauscht werden.
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]:
```suggestion
          Du kannst selbst ein Profilbild einstellen
```

**Comment by DanielHabenicht** [comment]
If you want to center the icons in the cards text e.g.: 
image

You can use `<button mat-icon-button disabled="true">`

File: Phonebook.Frontend/src/styles/mixins.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+//Custom Mixins
+@mixin mat-icon-size($size) {
+  font-size: $size;
+  height: $size;
+  width: $size;
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/styles/mixins.scss]: please add `line-height: $size` aswell

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.scss [hunk 6]
```diff
@@ -53,6 +53,10 @@ mat-card {
       @include mat-icon-size(60px);
     }
   }
+
+  mat-card-title {
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.scss]: Can you move this into the mat-card style section above?

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

**Comment by DanielHabenicht** [comment]
Preview Environment ready at https://137.135.135.208

**Pull request merged**